include/cneuron/cneuron.h

```diff
@@ -4,6 +4,8 @@
 #include <stdbool.h>
 #include <stddef.h>
 
+#include "../src/arena.h"
```

Comment:
```suggestion
#include "arena.h"
```
Update the CMake to include the src directory for headers in the apply_target function.
```
target_include_directories(${target} PRIVATE ${CMAKE_SOURCE_DIR}/src)
```